packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Role Assignment
+---
+<ApiLifecycle access="beta" />
+
+# Role assignment
+Role assignment to principals makes them administrators of your org. Principals can be Users or Groups of Users. When a Role is assigned to a Group, all members of the Group automatically have the privileges granted by the Role.
```

Comment:
Combine the first two sentences > "Users or groups of users who are assigned the role of principals become administrators of your org."

Reply:
This doesn't conceptually make much sense, especially when saying "who are assigned the role of principals"

Reply:
OK.